plugin/src/main/java/com/denizenscript/denizen/events/block/BrewsScriptEvent.java

```diff
@@ -25,11 +28,22 @@ public class BrewsScriptEvent extends BukkitScriptEvent implements Listener {
     // @Context
     // <context.location> returns the LocationTag of the brewing stand.
     // <context.inventory> returns the InventoryTag of the brewing stand's contents.
+    // <context.fuel_level> returns an ElementTag(Number) of the brewing stand's fuel level.
+    // <context.result> returns a ListTag(ItemTag) of the brewing stand's result.

```

Comment:
Would be nice to have the context's meta match the determination's meta, e.g. `the items that will be brewed`/whatever you make the determination meta